### ISSUE TITLE
It's not formatting properly lines that contain regexp

### ISSUE THREAD

**Issue by tifDev** [sumnow/markdown-formatter#22]
I have a markdown files with some regexp and when i format, it does the following behaviour:
Before formatting:
```
# Regexp
## Regex A
`^(.*myPattern.*)$`
## Regex B
`([^\/]+)$`
```
After formatting:
```
# Regexp
## Regex A
`^(.*myPattern.*)# Regexp
## Regex B
`([^\/]+)# Regexp
```
It seems to be removing some characters and appending the Heading 1.

You can test it, just open a .md file paste that and then format.
Congratulations for your extension, i found it very useful.

**Comment by tifDev** [comment]
It seems that only happens when the code is between ``, when i use code block like this:
```
code block
```
It formats properly.

**Comment by sumnow** [comment]
I'm sorry about this issue.
The problem has been fixed in the [0.8.0](https://github.com/sumnow/markdown-formatter/blob/master/CHANGELOG.md#080) version. Thank you.
